openhtf/io/output/mfg_inspector.py

```diff
@@ -444,4 +453,31 @@ def __call__(self, test_record):  # pylint: disable=invalid-name
     except UploadFailedError:
```

Comment:
This shouldn't catch the exception anymore, the framework will log the exception for you. That will also allow you to catch it in `UploadOrOutput`